assets/src/edit-story/components/colorPicker/currentColorPicker.js

```diff
@@ -165,12 +173,15 @@ function CurrentColorPicker({ rgb, hsl, hsv, hex, onChange, showOpacity }) {
       </Body>
       <Footer>
         {/* TODO: implement (see https://github.com/google/web-stories-wp/issues/262) */}
-        <EyedropperButton
-          width={EYEDROPPER_ICON_SIZE}
-          height={EYEDROPPER_ICON_SIZE}
-          aria-label={__('Select color', 'web-stories')}
-          isDisabled
-        />
+        {enableEyeDropper ? (
```

Comment:
The layout of the hex input field appears to be a bit off-center without the button.

Comparing your own screenshots:

![with eyedropper](https://user-images.githubusercontent.com/570239/91525017-31ae0400-e91e-11ea-8573-6d12495701e0.png)
![without eyedropper](https://user-images.githubusercontent.com/570239/91525045-3e325c80-e91e-11ea-92e6-1af8954856c7.png)